keel-ec2-plugin/src/test/kotlin/com/netflix/spinnaker/keel/ec2/resource/ClusterHandlerTests.kt

```diff
@@ -461,10 +464,69 @@ internal class ClusterHandlerTests : JUnit5Minutests {
             )
           }
         }
+
+        test("the default deploy strategy is used") {
+          runBlocking {
+            upsert(resource, diff)
+          }
+
+          val slot = slot<OrchestrationRequest>()
+          coVerify { orcaService.orchestrate("keel@spinnaker", capture(slot)) }
+
+          expectThat(slot.captured.job.first()) {
```

Comment:
Should we use the defaults from the class here so the test won’t break if they ever change?

Reply:
Yes, we should. Lazy of me.